employee-onboarding-A2A-reference-implementation/agents/orchestrator/graph.py

```
@@ -0,0 +1,510 @@
+        # Record skipped task
+        task_result = {
+            "step": current_task["step"],
+            "agent": current_task["agent_name"],
+            "task": current_task["task"],
+            "result": f"⏭️ SKIPPED - Approval denied: {denied_approval['reason'][:100]}",
+            "success": False,
+            "skipped": True
+        }
+        
+        task_results = state.get("task_results", [])
+        task_results = task_results + [task_result]
```

Ruff (0.15.10)

[warning] Consider `[*task_results, task_result]` instead of concatenation

Replace with `[*task_results, task_result]`

(RUF005)